front/app/components/LandingPages/citizen/EventsWidget.tsx

```diff
@@ -51,7 +51,6 @@ const Header = styled.div`
 
 const StyledTitle = styled(Title)`
   ${media.phone`
-    text-align: center;
```

Comment:
Not related to this, but just wanted to take it along: this title was centered on mobile which looked really bad so I removed the centering